index.html

```diff
@@ -59,5 +59,218 @@ <h3>LET'S CONNECT</h3>
+          <img src="/images/work4.jpg" alt="Work 1" />
+          <h1>Multi-Post <br />Stories</h1>
+        </div>
+
+        <div class="item2">
+          <p class="canopy">CANOPY</p>
+          <span class="point"></span>
+          <p>Back End Dev</p>
+          <span class="point"></span>
+          <p>2015</p>
+        </div>
+
+        <div class="item3">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+        </div>
+
+        <div class="item4">
+          <ul>
+            <li>Html</li>
+            <li>Css</li>
+            <li>Javascript</li>
+          </ul>
+        </div>
+
+        <div class="item5">
+          <button>See project</button>
+        </div>
+      </container>
+    </div>
+
+    <div class="about">
+      <h1 class="about-title">About Myself</h1>
+      <p class="about-description">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website. Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hesitate to
+        contact me.
+      </p>
+
+      <div class="media">
+        <h3>LET'S CONNECT</h3>
+        <div class="socials">
+          <ul>
+            <li class="icon">
+              <a href="https://github.com/peter-mwovi/"
+                ><img src="/images/github.png" alt="github link"
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="https://www.linkedin.com/in/peter-mwovi-57141a179/"
+                ><img src="/images/Linkedin_icon.png" alt="linked link"
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="#"><img src="/images/doits.png" alt="doit" /></a>
+            </li>
+            <li class="icon">
+              <a href="https://twitter.com/mwovi_peter"
+                ><img src="/images/twitter.png" alt="twitter link"
+              /></a>
+            </li>
+            <li class="icon">
+              <a href="#"><img src="/images/M.png" alt="M" /></a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="resume">
+          <button>Get My Resume</button>
+        </div>
+      </div>
+
+      <details>
+        <summary>
+          <h1>Languages</h1>
+        </summary>
+        <div class="option1">
+          <img src="/images/js.png" alt />
+          <p>JavaScript</p>
+        </div>
+        <div class="option2">
+          <img src="/images/html.png" alt />
+          <p>Html</p>
+        </div>
+        <div class="option3">
+          <img src="/images/css.png" alt />
+          <p>Css</p>
+        </div>
+      </details>
+      <hr />
+      <details>
+        <summary>
+          <h1>Framework</h1>
+        </summary>
+      </details>
+      <hr />
+
+      <details>
+        <summary>
+          <h1>Skills</h1>
+        </summary>
+      </details>
```

Comment:
- [ ] Please consider to use `ul` to wrap the list of  The technology in each category ("Languages", "Frameworks" and "Skills") and make sure each of them is `li`. This will make your code way more cleaner 🧼 and improve the user experience 👍

Reply:
This issue has been resolved.